src/EditorFeatures/Core/DocumentationComments/CopilotGenerateDocumentationCommentProvider.cs

```diff
@@ -46,14 +47,14 @@ public async Task InitializeAsync(ITextView textView, SuggestionServiceBase sugg
         public async Task GenerateDocumentationProposalAsync(DocumentationCommentSnippet snippet,
             ITextSnapshot oldSnapshot, VirtualSnapshotPoint oldCaret, CancellationToken cancellationToken)
         {
-            await Task.Yield().ConfigureAwait(false);
+            await YieldAwaitableExtensions.ConfigureAwait(Task.Yield(), false);
```

Comment:
```suggestion
            await TaskScheduler.Default;
```

you'll also need `using Microsoft.VisualStudio.Threading;`